src/Stratis.Bitcoin.IntegrationTests/Miners/ProofOfWorkMiningTests.cs

```diff
@@ -70,12 +73,15 @@ public void MiningAndPropagatingPOW_MineBlockNotPushedToConsensusCode_Superceded
 
                 // Nodes 1 syncs with node 2 up to height 7.
                 TestHelper.ConnectAndSync(node1, node2);
+                Assert.True(node1.FullNode.ConsensusManager().Tip.Height == 7);
 
                 // Call BlockMinedAsync manually with the block that was supposed to have been submitted at height 6.
                 node1.FullNode.ConsensusManager().BlockMinedAsync(manualBlock).GetAwaiter().GetResult();
 
                 // Verify that the manually added block is NOT in the consensus chain.
-                Assert.False(node1.FullNode.Chain.Contains(manualBlock.GetHash()));
+                var chainedHeaderToValidate = node1.FullNode.ConsensusManager().Tip.Previous;
+                Assert.True(chainedHeaderToValidate.Height == 6);
+                Assert.False(chainedHeaderToValidate.HashBlock == manualBlock.GetHash());
```

Comment:
The Assert wasn't accurate (or correct) before. We need to ensure that the manually created block at height 6 is NOT the true validated header at height 6 in the chain. 

The test creates a chain up until height 5, then we create a block at height 6 but dont submit it to consensus, then we mine 2 more blocks (6,7) and then try and submit the manual block at height 6. This will fail as the chain work is already higher now (7). Therefore the assert should check that he manual block is not at height 6 but the mined block is.